linuxNew/jdk/suse/src/main/packaging/temurin/17/temurin-17-jdk.spec

```diff
@@ -1,9 +1,9 @@
-%global upstream_version 17.0.2+8
+%global upstream_version 17.0.3+7
 # Only [A-Za-z0-9.] allowed in version:
 # https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version
 # also not very intuitive:
 #  $ rpmdev-vercmp 17.0.1.0.1___17.0.1.0+12
-#  17.0.1.0.0___12 == 17.0.1.0.0+12
+#  17.0.1.0.0___12 == 17.0.3.0.0+7
 %global spec_version 17.0.2.0.0.8
```

Comment:
@Haroon-Khel should this spec_version be changed as well? I see it changed in the redhat spec above

Reply:
Youre right. I changed the wrong line by mistake